lacci/lib/shoes/app.rb

```diff
@@ -333,22 +361,38 @@ def border(...)
   # Shape DSL methods
 
   def move_to(x, y)
-    raise(Shoes::Errors::InvalidAttributeValueError, "Pass only Numeric arguments to move_to!") unless x.is_a?(Numeric) && y.is_a?(Numeric)
-
-    if current_slot.is_a?(::Shoes::Shape)
-      current_slot.add_shape_command(["move_to", x, y])
+    unless x.is_a?(Numeric) && y.is_a?(Numeric)
+      raise(Shoes::Errors::InvalidAttributeValueError,
+            'Pass only Numeric arguments to move_to!')
     end
+
+    return unless current_slot.is_a?(::Shoes::Shape)
+
+    current_slot.add_shape_command(['move_to', x, y])
   end
 
   def line_to(x, y)
-    raise(Shoes::Errors::InvalidAttributeValueError, "Pass only Numeric arguments to line_to!") unless x.is_a?(Numeric) && y.is_a?(Numeric)
-
-    if current_slot.is_a?(::Shoes::Shape)
-      current_slot.add_shape_command(["line_to", x, y])
+    unless x.is_a?(Numeric) && y.is_a?(Numeric)
+      raise(Shoes::Errors::InvalidAttributeValueError,
+            'Pass only Numeric arguments to line_to!')
     end
+
+    return unless current_slot.is_a?(::Shoes::Shape)
+
+    current_slot.add_shape_command(['line_to', x, y])
   end
 
   # Not implemented yet: curve_to, arc_to
 
-  alias_method :info, :puts
+  alias info puts
+
+  private
+
+  def render_index_if_defined_on_first_boot
+    return if $first_boot_finished
```

Comment:
Maybe a global here is too gnarly and an ivar will do

Reply:
Yeah, ivar would be plenty.